### PR TITLE
Cayenne

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/cayenne/cayenne/accelerometer.schema.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  "title": "Accelerometer",
+  "type": "object",
+  "properties": {
+    "channel": {
+      "title": "Channel",
+      "type": "number",
+      "description": "Channel number",
+      "hideFromKpis": true
+    },
+    "xAxisAccelerometer": {
```
Comment (mariusgiger) [inline types/cayenne/cayenne/accelerometer.schema.json:13]: please align the naming with other accelerometers

File: types/cayenne/cayenne/gyrometer.schema.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "title": "Gyrometer",
+  "type": "object",
+  "properties": {
+    "channel": {
+      "title": "Channel",
+      "type": "number",
+      "description": "Channel number",
+      "hideFromKpis": true
+    },
+    "gyroX": {
```
Comment (mariusgiger) [inline types/cayenne/cayenne/gyrometer.schema.json:13]: align this naming with other gyrometers

File: types/cayenne/cayenne/meta.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  "name": "Cayenne",
+  "version": "1.0.0",
+  "manufacturer": "Cayenne",
+  "url": "https://docs.mydevices.com/docs/lorawan/cayenne-lpp",
+  "description": "The LPN CM-1 has a built in temperature and humidity sensor.",
+  "author": "Akenza AG",
+  "firmwareVersion": "V0.0.7",
+  "loraDeviceClass": "A",
+  "availableSensors": ["Humidity", "Temperature", "Button"],
+  "outputTopics": ["button_pressed", "default", "lifecycle"],
```
Comment (mariusgiger) [inline types/cayenne/cayenne/meta.json:11]: add missing topics

**Review by mariusgiger — CHANGES_REQUESTED** [review]
please align the schema with other types

File: types/cayenne/cayenne/gyrometer.schema.json (hunk 2 above)
Comment (J2thatsme) [reply types/cayenne/cayenne/gyrometer.schema.json:13]: I do think thats the standard till now

**Pull request merged**